auspice-client/customisations/splash.js

```diff
@@ -105,7 +105,7 @@ function ErrorMessage({errorMessage}) {
       There seems to have been an error accessing that dataset.
       <p style={{fontSize: "14px"}}>
         {`Details: ${errorMessage}. Please `}
-        <a href={"mailto:hello@nextstrain.org"} style={{color: "inherit", textDecoration: "underline"}}>
+        <a href={"mailto:support@example.com"} style={{color: "inherit", textDecoration: "underline"}}>
```

Comment:
Good point. I mostly made this change to note where auspice-client should be made customizable. I've identified some more points in e49ad0059aced63d7ef5d262fdb24bda11f56aef.

I'm hoping it can be made customizable, but if that turns out to be too difficult, I'll leave it as-is.

Reply:
See https://github.com/nextstrain/nextstrain.org/pull/735#discussion_r1397949502